com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/helpers/MtaArchiveBuilderTest.java

```diff
@@ -47,6 +47,7 @@
              }, 
              // (1) testCannotReadDeploymentDescriptor
              {
+                 //TODO ask what should code do when someone is trying to use version 1
```

Comment:
fail with a friendly message advising the user to migrate to the latest spec version